R/question_parsers.R

```diff
@@ -53,64 +60,81 @@ parse_question_info <- function(ques){
   # join them
   # then will join with responses on unique ID (Q/row/col) & choice_id
   out <- q_info
-  if(!is.null(rows)) { out <- merge(out, rows) }
-  if(!is.null(cols)) { out <- merge(out, cols) }
-  if(!is.null(choices)) { out <- merge(out, choices) }
-  if(!is.null(other)) {
-    out <- dplyr::bind_rows(out, other)%>%
-      tidyr::fill(heading:question_subtype)
+  if (!is.null(rows)) {
+    out <- merge(out, rows)
+  }
+  if (!is.null(cols)) {
+    out <- merge(out, cols)
+  }
+  if (!is.null(choices)) {
+    out <- merge(out, choices)
+  }
+  if (!is.null(other)) {
+    out <- dplyr::bind_rows(out, other) %>%
+      tidyr::fill(.data$heading:.data$question_subtype)
   }
 
   tibble::as_tibble(out)
 }
 
 ## These functions below are called by parse_question_info
 
+#' @importFrom rlang .data
```

Comment:
Probably my ignorance - should the `@importFrom rlang .data` be here twice.

Reply:
I don't think it is actually necessary no. I think my motivation was that you have a function definition inside a function definition. Which personally I would be looking to refactor out if possible. feel free to remove if you like